src/SyntaxHighlighter/Cpp.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SyntaxHighlighter/Cpp.h"
+
+#include <qregularexpression.h>
+
+#include <QColor>
+#include <QString>
+#include <QTextCharFormat>
+
+namespace orbit_syntax_highlighter {
+namespace C {
```

Comment:
It's fine to put these constants into a namespace with a name. But you still need the anonymous namespace as well.

Reply:
Thanks! Do you like it now?

Reply:
Yes. Looks good!